tools/csv-merger/csv-merger.go

```diff
@@ -502,14 +511,7 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 	// Add tolerations to deployments
 	for i := range templateStrategySpec.DeploymentSpecs {
 		d := &templateStrategySpec.DeploymentSpecs[i]
-		d.Spec.Template.Spec.Tolerations = []corev1.Toleration{
-			{
-				Key:      "node.ocs.openshift.io/storage",
-				Operator: corev1.TolerationOpEqual,
-				Value:    "true",
-				Effect:   corev1.TaintEffectNoSchedule,
-			},
-		}
+		d.Spec.Template.Spec.Tolerations = ocsNodeToleration
```

Comment:
We should move the entire loop below i.e. after we append metrics-exporter deployment to templateStrategySpec. That should give the same effect without code duplication.

Reply:
...oh, duh! Changed.